src/main/java/bitmap/transformer/App.java

```diff
@@ -16,14 +16,14 @@ public String getGreeting() {
     }
 
     public static void main(String[] args) {
-        readBitMap();
         System.out.println(new App().getGreeting());
```

Comment:
You can safely delete the greeting stuff once you've verified your project was set-up and running.